Project02Eclipse/src/com/hemebiotech/analytics/WriteSymptomDataResult.java

```diff
@@ -0,0 +1,36 @@
+package com.hemebiotech.analytics;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+
+public class WriteSymptomDataResult implements ISymptomResultWriter{
+    private final FileWriter writer = new FileWriter("result.out");
```

Comment:
This needs to be instanced in the constructor, the "result.out" will be injected when the object is initialized.